examples/image-processing/log.py

```diff
@@ -0,0 +1,17 @@
+from sys import stdout
+
+from loguru import logger
```

Comment:
Maybe not worth it to use a third-party logger for a simple example? Let's stick to the standard library.

Reply:
i also changed the loguru with the standard logging library.